### PR TITLE
feat: #1596 Implement email styling enhancements

### PR DESCRIPTION
**Pull request by ianliuwk1019**
Closes: #1596 
* update frontend T&C component to add some note.
* on GC Notify, create _V2 templates for both regular user template and delegated admin template.
* update both "server/backend" and "server/admin_management" on GCNotifyEmailService to 
  reflect the template id change and params change.
* update dynamic wording in code for templates parameters.
* save latest version to project "public" folder to be accessible as a link, and use it for delegated admin template.
* update granting user and granting delegated admin method to provide data for email params.
* adjust email template to show scoped client as a point lists.
* minor other email template adjustments on wording and spacing.

### CODE REVIEW

File: server/admin_management/api/app/utils/utils.py [hunk 1]
```diff
@@ -28,3 +29,10 @@ def remove_app_env_suffix(name: str):
         if name.endswith(suffix):
             return name[: -len(suffix)]
     return name
+
+
+def is_success_response(response: Response):
+    SUCCESS_LIST = [
+        HTTPStatus.OK, HTTPStatus.CREATED, HTTPStatus.ACCEPTED, HTTPStatus.NO_CONTENT
+    ]
```
Comment (craigyu) [inline server/admin_management/api/app/utils/utils.py:37]: I prefer to have this in the constants file, but this is fine too

**Review by craigyu — APPROVED** [review]
Good work, i didn't test it locally since it was tested already

File: server/admin_management/api/app/utils/utils.py (hunk 1 above)
Comment (ianliuwk1019) [reply server/admin_management/api/app/utils/utils.py:37]: Thanks. This I don't feel it will be used by other, for now just here.

**Comment by ianliuwk1019** [comment]
> Good work, i didn't test it locally since it was tested already

No problem, definitely need to be tested once it is merged to dev.

**Pull request merged**